lib/ansible/modules/rpm_key.py

```diff
@@ -164,30 +167,42 @@ def fetch_key(self, url):
         tmpfile.close()
         return tmpname
 
+    def normalize_keyids(self, keyids):
+        normalized_keyids = []
+        for keyid in keyids:
+            normalized_keyids.append(self.normalize_keyid(keyid))
+
+        return normalized_keyids
+
     def normalize_keyid(self, keyid):
         """Ensure a keyid doesn't have a leading 0x, has leading or trailing whitespace, and make sure is uppercase"""
         ret = keyid.strip().upper()
-        if ret.startswith('0x'):
+        if ret.startswith("0x"):
             return ret[2:]
-        elif ret.startswith('0X'):
+        elif ret.startswith("0X"):
             return ret[2:]
         else:
             return ret
 
-    def getkeyid(self, keyfile):
+    def getkeyids(self, keyfile):
         stdout, stderr = self.execute_command([self.gpg, '--no-tty', '--batch', '--with-colons', '--fixed-list-mode', keyfile])
+        keyids = []
         for line in stdout.splitlines():
             line = line.strip()
             if line.startswith('pub:'):
-                return line.split(':')[4]
+                keyids.append(line.split(':')[4])
+
+        if keyids == []:
```

Comment:
This is what people usually expect in Python:
```suggestion
        if not keyids:
```